### ISSUE TITLE
Elon Musk: F*ck that, we'll get it done | Lex Fridman Podcast Clips by Lex Clips

### ISSUE THREAD

**Issue by trainfarb** [hu3agency/LP0_polymathProcess#42]

**Comment by Patrick-T-Anderson** [comment]
In the phrase "we'll get **it** done",
there are many goals to consider.

Which **it** is the most important?

Do we really need to rocket away,
while so many struggle on Earth?

Will there be homeless on Mars?

   _{{SONG}}_

Everyone needs land,
and a way to **gitter-done**.
...
We need a better way to organize!
...
the production that we need,
for the food and the housing,
and most that we usually buy.
...
But when _Users_ own _Sources_ because they need _Goods_,
and _Users_ trade _Work_ because they need _Services_,
production can be Free as in Freedom.
...
Rent is zero (0) when owner occupied.
When you own bees, you own honey.

**Comment by trainfarb** [comment]
@Patrick-T-Anderson Agreed! Thanks for your comment. We're using his responses (and responses from other in the future) to create memes that answer questions related to huStory. We're applying "getting it done" to whatever challenges you laid out.

The purpose of this issue is to help people realize that we don't need to criticize Elon Musk and tell him what he should be doing with his mind. The purpose is to remind every person that we all have the ability to think of a challenge in the world and learn what we want to "get it done"

Love the ideas you put out...we'll need to get you playing this game!
